### PR TITLE
gateway-remote

### PR DESCRIPTION
**Pull request by blockchainjoshua**

### CODE REVIEW

File: B827EBFFFE937A49.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "eui-b827ebfffe937a49",
```
Comment (gonzalocasas) [inline B827EBFFFE937A49.json]: Please use the upper cased EUI, without the `eui-` prefix, i.e.:

```suggestion
		"gateway_ID": "B827EBFFFE937A49", 
```

File: B827EBFFFE937A49.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "eui-b827ebfffe937a49", 
+		"servers": [ 
+			{ 
+				"server_address": "asia-se.thethings.network",
```
Comment (gonzalocasas) [inline B827EBFFFE937A49.json]: This is not the router address, please check here for the list of valid addresses: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

(I guess you want `router.as1.thethings.network` or `router.as2.thethings.network`)

File: B827EBFFFEF94ED9.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "eui-b827ebfffef94ed9",
```
Comment (gonzalocasas) [inline B827EBFFFEF94ED9.json]: All same comments apply for this file as well.

File: B827EBFFFE937A49.json (hunk 2 above)
Comment (blockchainjoshua) [reply B827EBFFFE937A49.json]: ok thanks

Comment (blockchainjoshua) [reply B827EBFFFE937A49.json]: "server_address": "router.as1.thethings.network",

**Comment by gonzalocasas** [comment]
One of the two files is fixed now, but the second one still has the same issues mentioned above.

**Comment by gonzalocasas** [comment]
ping...

**Comment by gonzalocasas** [comment]
Please fix the second file as well, otherwise we'll close this in the coming days if there's no activity.

**Comment by gonzalocasas** [comment]
Closing due to inactivity. Feel free to re-open once you find the time to continue on this.

**Pull request closed without merging**